contracts/storage/LibAppStorage.sol

```diff
@@ -39,6 +39,7 @@ struct AppStorage {
     mapping(bytes32 => address) doubleHashedShardArchaeologists;
 
     // sarcophagus ids
+    // todo: is this used?
```

Comment:
I highly doubt it. Seth and Kyle may have more context on if there was some future plans for having this